components/authn-service/tokens/pg/adapter.go

```diff
@@ -55,6 +55,7 @@ func (a *adapter) CreateLegacyTokenWithValue(ctx context.Context, value string)
 
 // PurgeProject removes a project from every token it exists in
 func (a *adapter) PurgeProject(ctx context.Context, projectID string) error {
+	a.logger.Info("starting project delete for id " + projectID)
```

Comment:
Added consistent logging so we can tell what is going on with this multi-service operation. It is infrequent enough and complex enough that `info` level is warranted.